references/segmentation/train.py

```diff
@@ -36,11 +39,12 @@ def get_transform(train, args):
         weights = torchvision.models.get_weight(args.weights)
         trans = weights.transforms()
 
-        def preprocessing(img, target):
+        def preprocessing(sample):
+            img, target = sample
             img = trans(img)
             size = F.get_dimensions(img)[1:]
             target = F.resize(target, size, interpolation=InterpolationMode.NEAREST)
-            return img, F.pil_to_tensor(target)
+            return img, F.to_image_tensor(target)
```

Comment:
Oops, this doesn't look right. This will return target as Image.
```suggestion
            return img, F.pil_to_tensor(target)
```

Reply:
Potential fix at https://github.com/pytorch/vision/pull/6588